src/framework/security/services/acl.service.ts

```diff
@@ -97,11 +97,8 @@ export class NbAclService {
     this.validateResource(resource);
 
     const parentRole = this.getRoleParent(role);
-    let parentCan = false;
-    if (parentRole) {
-      parentCan = this.can(this.getRoleParent(role), permission, resource);
-    }
-    return parentCan ? parentCan : this.exactCan(role, permission, resource);
+    const parentCan = parentRole && this.can(this.getRoleParent(role), permission, resource);
```

Comment:
You could use `parentRole` again here:
`const parentCan = parentRole && this.can(parentRole, permission, resource);`